iocore/net/SSLCertLookup.cc

```diff
@@ -404,6 +389,7 @@ SSLContextStorage::lookup(const char *name) const
 {
   InkHashTableValue value;
 
+  // First look for an exact name match
```

Comment:
Does the lookup do case insensitive search? Or is the input guaranteed to be lower case?